charts/openstack-cloud-controller-manager/templates/daemonset.yaml

```diff
@@ -25,6 +25,8 @@ spec:
       labels:
         {{- include "occm.controllermanager.labels" . | nindent 8 }}
     spec:
+      imagePullSecrets:
+        {{- toYaml .Values.imagePullSecrets | nindent 8 }}
```

Comment:
It might be an idea to enclose this in a `with`  block, similar to other settings in the chart.

This would look similar to:

```
{{- with .Values.imagePullSecrets }}
imagePullSecrets:
  {{- toYaml . | nindent 8 }}
{{- end }}
```

Reply:
+1, makes sense.

Reply:
This was fixed, please re-check.